symfony/framework-bundle/4.2/src/Kernel.php

```diff
@@ -43,12 +43,12 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $loader->load($confDir.'/{services}_'.$this->environment.self::CONFIG_EXTS, 'glob');
     }
 
-    protected function configureRoutes(RouteCollectionBuilder $routes): void
+    protected function configureRouting(RoutingConfigurator $routes): void
```

Comment:
The changes in this file look wrong as this is for 4.2.

Reply:
fixed